docs/guides/admin/docs/upgrade.md

```diff
@@ -48,6 +48,20 @@ Please make sure to compare your configuration against the current configuration
   [this document](https://github.com/elan-ev/opencast-studio/blob/2020-09-14/CONFIGURATION.md).
 
 
+### Wowza streaming configuration changes
+
+In [pull request #1179](https://github.com/opencast/opencast/pull/1179), the configuration of Wowza servers was made
+tenant-specific. Before, you had to specify `org.opencastproject.streaming.url` and
+`org.opencastproject.streaming.port` in `etc/custom.properties`. Starting with Opencast 9, those values have to be
+specified as `org.opencastproject.<tenant>.wowza.url` and `org.opencastproject.<tenant>.wowza.port` in
+`etc/org.opencastproject.distribution.streaming.wowza.WowzaStreamingDistributionService.cfg`.
```

Comment:
You are right, we don't need to mention twice that it is now tenant-specific.
My main intention here is to not be as specific as you are. The configuration file is new and people should look at it and the documentation it contains. Your specific suggested change implies that users should ignore the file but for this key. Also, there is documentation about this in the file and in the docs.
So, I would simply go with:

```suggestion
In [pull request #1179](https://github.com/opencast/opencast/pull/1179), the configuration of Wowza servers was made
tenant-specific and has been moved to `etc/org.opencastproject.distribution.streaming.wowza.WowzaStreamingDistributionService.cfg`.
```

That should give users all information they need, shouldn't it?

Reply:
Fiiiiiiine. This strange desire to remove everything from docs that not strictly necessary :D 
I pushed. Should be fine now.